src/gluonts/torch/model/d_linear/module.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import Tuple
+
+import torch
+from torch import nn
+
+from gluonts.core.component import validated
+from gluonts.model import Input, InputSpec
+from gluonts.torch.distributions import StudentTOutput
+from gluonts.torch.scaler import StdScaler, MeanScaler, NOPScaler
+
+
+def make_linear_layer(dim_in, dim_out):
```

Comment:
Should we reuse the same method from [`simple_feedfoward/model.py`](https://github.com/awslabs/gluonts/blob/dev/src/gluonts/torch/model/simple_feedforward/module.py#L28)?